### DISCUSSION TITLE
Total Exp Formatted

### DISCUSSION THREAD

**Discussion by kura78** [Zrips/CMI#8091] [Feature Requests]
I was just wondering if %cmi_user_totalexp_formatted% and %cmi_user_totalexp_formated% could be added as %cmi_user_totalexp% already exists and it would be cool to have a formated version of this. Thanks!

**Comment by CloudeLecaw** [comment]
what these formated once should be shown?

we have so many exp placeholder allready....im not sure why more are needed.

%cmi_user_missingexp% – Missing exp amount until next level
%cmi_user_missingexpp% – Missing exp in percentage until next level
%cmi_user_exp% – Current exp amount for current level
%cmi_user_expp% – Current exp in percentage for current level
%cmi_user_totalexp% – Total amount of exp player have
%cmi_user_level% - Players level

**Reply by ihateblueb** [reply]
Total exp formatted would output, for example, 1,000,000 instead of 1000000 to make it easier on the eyes. this already exists for other placeholders.

**Reply by CloudeLecaw** [reply]
It also means it needs  a place in config where it is defined when a , (or . if wanted instead of ,) is used - like you need to define it in config for the moneyformat

So a placeholder would not be enough. it also needs a config part where you can define the "xpformat"


but at least i like this idea :-)
